schemas/com.snowplowanalytics.travel/content/jsonschema/1-0-0

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "type": "object",
+  "self": {
+    "vendor": "com.snowplowanalytics.travel",
+    "name": "content",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "description": "Entity to describe a piece of content",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the content item"
+    },
+    "location": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Location of the content item"
```

Comment:
This doesn't specify `(physical)` like the other descriptions for `location` fields, does that omission mean something?
Is the content like in a drawer or something, or located in a sidebar, is it content associated with a destination that has a location?
I'm not sure what to expect from or populate this field with.
What's if it's like a listicle with N destinations described?